plugins/content_types/loans.inc

```diff
@@ -126,10 +126,30 @@ function ding_loan_loans_form($form, &$form_state, $account, $loans = array()) {
         $title = l($entity->getTitle(), $uri['path']);
       }
 
+      // Add creator(s).
+      $creators = $entity->creators;
```

Comment:
This code seams to be the same as in https://github.com/ding2/ding_reservation/pull/10/files would it be possible to move this to an utility function or maybe into an pre-process in ding_base? As the information in both places is used by "material_item" theme function.